app/controllers/questions/questions_controller.rb

```diff
@@ -78,4 +78,4 @@ def form_class
       end
     end
   end
-end
```

Comment:
add analytics for successful posts and validation errors

Reply:
also add analytics for notification opt-ins and how often people change their address